graph/client-e2e/project.json

```diff
@@ -4,7 +4,17 @@
   "sourceRoot": "graph/client-e2e/src",
   "projectType": "application",
   "targets": {
+    "pre-e2e": {
+      "inputs": ["default", "^production", "{workspaceRoot}/jest.preset.js"],
+      "executor": "@nx/jest:jest",
```

Comment:
This could have been just a simple `node ...` script ran by `run-commands`. There was no need for a unit test.

Reply:
Unless we just call the `pnpm install cypress` command manually, this needs to be in a unit test, because the `ensureCypressInstallation` helper relies on `jest`.

We also probably cant just use `pnpm` install as we'd need to know the current package manager. Perhaps a script could find the PM and then install it? Would that be preferable?

<img width="1273" alt="image" src="https://github.com/nrwl/nx/assets/12140467/f29d2d12-30a6-4053-a4b6-92bbf4593dc9">